layouts/partials/footer.html

```diff
@@ -112,6 +112,7 @@ <h3 class="h6 text-white">Resources</h3>
           <li><a class="list-group-item list-group-item-action" href="/docs/">Documentation</a></li>
           <li><a class="list-group-item list-group-item-action" href="/docs/report-abuse/">Report abuse</a></li>
           <li><a class="list-group-item list-group-item-action" href="/docs/report-blocking-website/">Report blocking website</a></li>
+          <li><a class="list-group-item list-group-item-action" href="/docs/why-simplelogin-is-not-disposable-mail.md/">Why SimpleLogin is not disposable mail</a></li>
```

Comment:
hey this URL should rather be https://simplelogin.io/docs/why-simplelogin-is-not-disposable-mail/ I think